### ISSUE TITLE
Update age normalization method in sequence based model evaluator

### ISSUE THREAD

**Issue by xj2193** [cumc-dbmi/cehr-bert#6]
The current normalization method for age is applied on the entire dataset before splitting up which might cause age information leakage for the future so it isn't aligned with the best practice. The method needs to be updated to make the evaluations fair across train/test/validation sets.

For sequence model including LSTM and BERT based, the batch normalization could be used to track the running average in the training data used later in test and validation sets.

https://github.com/cumc-dbmi/cehr-bert/blob/master/models/evaluation_models.py
https://github.com/cumc-dbmi/cehr-bert/blob/master/evaluations/model_evaluators.py
